### ISSUE TITLE
"JWT token not found" as Json, instead of JsonLD

### ISSUE THREAD

**Issue by shehi** [api-platform/api-platform#1174]
In my api-platform implementation, I actually have everything working, with one small, nuisant part: whenever I try to access something which is behind JWT, I get response in JSON format:
```{"code":401,"message":"JWT Token not found"```
As a JSON it's alright, but I specifically ask for ```Accept: application/ld+json```. I'd expect having some line resembling this:
```"hydra:description":"Not authenticated!"```

So, any idea how I can force JWT to emit Json-LD response, instead of Json?

**Comment by dmitry-vinogradov** [comment]
I think generating of the response in this case is not in responsibility of the Api Platform. Please read [here](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/2-data-customization.md#eventsjwt_not_found---customizing-the-response-on-token-not-found) how you can customize the response.

**Comment by mrtnzagustin** [comment]
Exactly, you should use what @dmitry-vinogradov says -> https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/2-data-customization.md#eventsjwt_not_found---customizing-the-response-on-token-not-found, i recommed you to create a custom JSONLDResponse class to make this more modular. @soyuka or @dunglas yo should close this one.

**Comment by mrtnzagustin** [comment]
@shehi check this https://github.com/api-platform/core/blob/ce7fffbd6ef109749d577c2b001ac70ff60dea5a/src/Hydra/Serializer/ErrorNormalizer.php
